frontend/src/app/application-forms/fields/noncommercial-fields.component.html

```diff
@@ -13,7 +13,7 @@
         <label id="participants-label" class="usa-input">Estimated number of participants <span class="required-fields-asterisk">*</span></label>
         <span id="participants-hint-text" class="help-text usa-form-hint">Please provide the number of event participants (Note: The use of a dash is permitted to supply a possible range).</span>
       </div>
-      <input class ="number-input-sizing" id="participants" type="tel" pattern="[0-9\-]+" maxlength="7" (keypress)="numberOnly($event)" formControlName="numberParticipants" aria-required="true" min="1"
+      <input class ="number-input-sizing" id="participants" type="tel" pattern="[0-9\-]+" maxlength="7" formControlName="numberParticipants" aria-required="true" min="1"
```

Comment:
input type should still be "text". we're not looking for a tel input. but the regex pattern is correct and will handle correct inputs. tested locally to work.